.ci/config/stage-types.yaml

```diff
@@ -32,7 +32,9 @@ db-unit:
   - 'mariadb'
   - 'sqlserver'
   - 'postgresql'
-  jdkVersion: 'openjdk-jdk-11-latest'
+  podSpec:
```

Comment:
**Observation**: The podspec is propagated to the calculated stages under `db-unit` stage